corehq/apps/reports/view_helpers.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from functools import partial
+import datetime
+import numbers
+import pytz
+
+from couchdbkit import ResourceNotFound
+from django.urls import reverse
+from django.utils.translation import ugettext as _
+
+from casexml.apps.case.views import get_wrapped_case
+from corehq.apps.hqwebapp.templatetags.proptable_tags import get_display_data
+
+
+def case_hierarchy_context(case, options):
+
+    wrapped_case = get_wrapped_case(case)
+    get_case_url = options.get('get_case_url')
+    timezone = options.get('timezone', pytz.utc)
```

Comment:
you could maybe make these kwargs now to make it more explicit what this function is accepting

Reply:
Good point, and doing this made me realize not all of these options are used. 839aa26